src/main/java/TTT/peaksAndSummitsHandler/PeaksAndSummitsHandler.java

```diff
@@ -0,0 +1,69 @@
+package TTT.peaksAndSummitsHandler;
+
+import org.apache.poi.hssf.usermodel.HSSFCell;
+import org.apache.poi.hssf.usermodel.HSSFRow;
+import org.apache.poi.hssf.usermodel.HSSFSheet;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.poifs.filesystem.POIFSFileSystem;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.ArrayList;
+
+public class PeaksAndSummitsHandler {
+    public static ArrayList<Top> polishSummits = new ArrayList<>();
```

Comment:
It seems like you're trying to create a singleton here, but this is not how you do it